pkg/ctl/update/cluster.go

```diff
@@ -132,7 +132,7 @@ func doUpdateClusterCmd(cmd *cmdutils.Cmd) error {
 		logger.Critical("failed checking nodegroups", err.Error())
 	}
 
-	cmdutils.LogPlanModeWarning(cmd.Plan && (stackUpdateRequired || versionUpdateRequired))
+	cmdutils.LogPlanModeWarning(cmd.Plan)
```

Comment:
Thanks for making the changes :) 

I think that to keep the plan mode consistent across commands this should be left with the full condition as it was before. And because we fixed the other return value it should now print the message when it needs to.